README.md

```diff
@@ -18,7 +18,7 @@ Common operations include:
 - [reducing](#reducing): `accumulate`, `collapse`, and `reduce`
 
 ## Usage
-In order to use the available  itertools filters/functions via Twig, simply add this service definition in your `services.xml`
+In order to use the available itertools filters/functions via Twig, simply add this service definition in your `services.xml`
 ```php
 <service id="zicht_itertools_twig_extension" class="Zicht\Itertools\twig\Extension">
```

Comment:
Possible add configuration, i.e. allow renaming twig `it`, `itf`, `itm`, and `itr`.

Reply:
Is this still a plan? You could also introduce this as a new feature in later releases as it is not breaking...

Reply:
With the removal of `itf`, `itm`, and `itr` it would only make `it` configurable... I see little reason to do that now, if at some point it conflicts, we can easily add it as a feature then.